ReleaseProcessAutomation/Configuration/Data/Settings.cs

```diff
@@ -23,7 +23,10 @@
 namespace ReleaseProcessAutomation.Configuration.Data;
 
 public class Settings
-{
+{  
   [XmlElement("msBuildPath")]
   public string MSBuildPath { get; set; }
+  
+  [XmlElement("alternativeMSBuildPath")]
+  public string[] AlternativeMSBuildPath { get; set; }
```

Comment:
I just realized that we should use a plural here for both the XML name and the property name. Otherwise it is kind of strange to have multiple `<alternativeMSBuildPath>` next to each other